backend/experiment/rules/rhythm_battery_intro.py

```diff
@@ -118,11 +138,9 @@ def first_round(self, experiment):
                 Step(_("Please use headphones, and turn off sound notifications from other devices and applications (e.g., e-mail, phone messages)."),
                      )],
             step_numbers=True,
-            button_label=_('OK')
+            button_label=_('Ok')
         )
-        playlist = Playlist(experiment.playlists.all())
         return [
-            consent,
+            self.intro_explainer(),
```

Comment:
I think it would be nicer if you'd also initialise this as `intro_explainer = self.intro_explainer()` or something for consistency.